components/sst_importer/src/util.rs

```diff
@@ -50,11 +52,14 @@ pub fn prepare_sst_for_ingestion<P: AsRef<Path>, Q: AsRef<Path>>(
         // RocksDB must not have this file, we can make a hard link.
         fs::hard_link(path, clone)
             .map_err(|e| format!("link from {} to {}: {:?}", path, clone, e))?;
+        File::open(clone)?.sync_all()?;
     } else {
         // RocksDB may have this file, we should make a copy.
         copy_and_sync(path, clone)
             .map_err(|e| format!("copy from {} to {}: {:?}", path, clone, e))?;
     }
+    // sync clone dir
+    File::open(Path::new(clone).parent().unwrap())?.sync_all()?;
```

Comment:
return error instead of unwrap?

Reply:
Since clone is a valid file path in previous options, the parent() call to parse its directory should not fail here

Reply:
yeah, but I think we can avoid use of unwrap after all.